docs/contributing/release-process.stories.mdx

```diff
@@ -0,0 +1,31 @@
+import { Meta } from '../../.storybook/components';
+
+<Meta title="Introduction/Contributing/Release Process" />
+
+# Release Process
+
+Circuit UI follows semantic versioning. In short, this means we use patch versions for bugfixes, minor versions for new features, and major versions for breaking changes.
+
+## Changesets
+
+Circuit UI uses [changesets](https://github.com/atlassian/changesets) to do versioning. A changeset is a piece of information about changes made in a branch or commit. It holds three bits of information:
+
+- What needs to be released
+- What version the packages should be released at (using a [semver bump type](https://semver.org/))
+- A changelog entry for the released packages
+
+Refer to the [official documentation](https://github.com/atlassian/changesets/blob/main/docs/adding-a-changeset.md#i-am-in-a-multi-package-repository-a-mono-repo) for more information.
+
+## Release branches
+
+We have a couple of special branches that are used for stable releases and prereleases.
+
+- **`main`** - The code in the `main` branch is stable and production-tested. When a PR is merged to `main` that contains a new changeset, `changesets` opens a PR and keeps it up to date with the latest changes. When the PR is merged, a new version is automatically published to NPM and the changesets since the last release are added to CHANGELOG.md files in GitHub.
+- **`canary`** - This is a branch you can use to test your changes integrated with other branches before it is ready to be PR'd to `main`. It is also useful if you need to deploy the changes somewhere to test them. `canary` is a throw-away branch that can be recreated from `main` at any time. Usually, `canary` is a couple of commits ahead of `main`.
```

Comment:
The use case for canary isn't obvious to me if we merge to main. To test changes locally, we could also use `yarn add ./path-to-circuit`, couldn't we?

Reply:
OMG, you've just solved the local testing problem that I haven't been able to figure out! 🎉 I didn't know you could install packages from a file path 🤯

I only tried with `yarn link` which creates a symlink that breaks Node's module resolution and causes multiple versions of React and Emotion to be included, which in turn breaks the app. 

That still doesn't work with deployments to a testing environment though. In those cases, we'd still need a prerelease on NPM. I don't expect this to happen often though, as I mentioned above. 

I'll update the docs to include your `yarn add` trick!

Reply:
Sounds good, makes sense to keep canary for now 👍 

I would think that it's fine not to have a prerelease branch in the future, since apps using circuit could test changes on their own test environments while they upgrade, but let's see if it proves useful 🙂